modules/video/src/lib/gifshot/gifshot.ts

```diff
@@ -18,12 +18,13 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 var utils = {
   URL: globalThis.URL || globalThis.webkitURL || globalThis.mozURL || globalThis.msURL,
   getUserMedia: (function () {
+    if (!globalThis.navigator) return globalThis.navigator;
```

Comment:
This is a bit long... I'd prefer defining a helper function below and call it, rather than an IIFE